Lib/traceback.py

```diff
@@ -733,7 +733,11 @@ def __init__(self, exc_type, exc_value, exc_traceback, *, limit=None,
         # Capture now to permit freeing resources: only complication is in the
         # unofficial API _format_final_exc_line
         self._str = _safe_string(exc_value, 'exception')
-        self.__notes__ = getattr(exc_value, '__notes__', None)
+        try:
+            self.__notes__ = getattr(exc_value, '__notes__', None)
+        except Exception as e:
+            self.__notes__ = [
+                f'Ignored error getting __notes__: {_safe_string(e, '__notes__', repr)}']
```

Comment:
```suggestion
                f'Ignored error getting __notes__: {_safe_string(e, \'__notes__\', repr)}']
```

Thanks a lot @vsajip for handling the conflicts here. I think we need to do this for python3.11